Resources/Prototypes/Entities/Objects/Consumable/Food/snacks.yml

```diff
@@ -370,6 +370,21 @@
         - ReagentId: Theobromine
           Quantity: 3
 
+- type: entity
+  id: FoodSnackJellyBeans
+  parent: FoodSnackBase
+  name: jelly beans
+  description: Packet full of rainbow colored jelly beans.
+  components:
+  - type: FlavorProfile
+    flavors:
+    - bee
```

Comment:
Why do they taste like bees?

Reply:
how do you know jellybeans dont taste like bees?

Reply:
(unbeelivable sounded cool to me) fixed.